crates/nu-cmd-lang/src/core_commands/describe.rs

```diff
@@ -46,6 +46,19 @@ impl Command for Describe {
             detailed: call.has_flag(engine_state, stack, "detailed")?,
             collect_lazyrecords: call.has_flag(engine_state, stack, "collect-lazyrecords")?,
         };
+        if options.collect_lazyrecords {
+            nu_protocol::report_error_new(
+                engine_state,
```

Comment:
I didn't add the warning for `run_const`, since `report_error_new` needs an engine state.